common/src/test/java/com/microsoft/identity/common/AccessTokenTest.java

```diff
@@ -24,19 +24,33 @@
 
 import com.microsoft.identity.common.internal.dto.AccessTokenRecord;
 
-import junit.framework.Assert;
-
 import org.junit.Test;
 
 import java.util.Calendar;
 
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
 public class AccessTokenTest {
 
     @Test
     public void testExpiry() {
         final AccessTokenRecord accessToken = new AccessTokenRecord();
         accessToken.setExpiresOn(getCurrentTimeStr());
-        Assert.assertTrue(accessToken.isExpired());
+        assertTrue(accessToken.isExpired());
+    }
+
+    @Test
+    public void testShouldRefreshWhenSet() {
+        final AccessTokenRecord accessToken = new AccessTokenRecord();
+        accessToken.setRefreshOn(getCurrentTimeStr());
+        assertTrue(accessToken.shouldRefresh());
+    }
+
+    @Test
+    public void testShouldRefreshWhenNotSet() {
```

Comment:
Is this test named correctly? The naming of the method, paired with `assertFalse` looks inverted

Reply:
This looks like it may throw a `NullPointerException` with the fallback logic merged in b74482c298bcf0d5f06cd2816d7f3e7ed1422df2 if all properties are uninitialized... I don't expect this should be possible in the real world... may be good to be a little defensive here though if easy/cheap